V/VkFFTCUDALib/build_tarballs.jl

```diff
@@ -0,0 +1,87 @@
+using BinaryBuilder, Pkg
+using BinaryBuilderBase
+include(joinpath(@__DIR__, "..", "..", "fancy_toys.jl"))
+include(joinpath(@__DIR__, "..", "..", "platforms", "cuda.jl"))
+
+name = "VkFFTCUDALib"
+version = v"0.1.1"
+
+# Collection of sources required to complete build
+sources = [
+		   GitSource("https://github.com/PaulVirally/VkFFTCUDALib.git", "9c8cf2eae261b363e185e5043599ca2726f5aafd"),
+		   FileSource("http://us.download.nvidia.com/XFree86/Linux-x86_64/410.66/NVIDIA-Linux-x86_64-410.66.run", "8fb6ad857fa9a93307adf3f44f5decddd0bf8587a7ad66c6bfb33e07e4feb217"),
+		  ]
```

Comment:
Indentation in this file is very inconsistent, there's a mix of tabs and spaces.  Sticking to only spaces (4 spaces for each level) would be better
```suggestion
sources = [
    GitSource("https://github.com/PaulVirally/VkFFTCUDALib.git", "9c8cf2eae261b363e185e5043599ca2726f5aafd"),
    FileSource("http://us.download.nvidia.com/XFree86/Linux-x86_64/410.66/NVIDIA-Linux-x86_64-410.66.run", "8fb6ad857fa9a93307adf3f44f5decddd0bf8587a7ad66c6bfb33e07e4feb217"),
]
```

Reply:
Yes you're right... I was copying code between two different servers which had different editor configs and got some of my indentation messed up :D The new commits should resolve this.